packages/eslint-config-twilio/test/code/testFile.js

```diff
@@ -1,7 +1,7 @@
 module.exports = function uglyCode() {
-    return (
-    <div>
-    Hello World
-    </div>
-    );
+  return (
+  <div>
```

Comment:
This indentation is wrong - was this linted with our linter?

Reply:
Good catch. This file is used in a unit test but that fails because it doesn't support jsx. I am now wondering if this test should be changed so it really checks the linter instead of the parsing error but that should be something for a new PR.